lib/ref_arch_setup/install.rb

```diff
@@ -104,16 +356,17 @@ def upload_pe_conf(src_pe_conf_path = "#{RAS_FIXTURES_PATH}/pe.conf",
     #
     # @param [string] src_pe_tarball_path Path to the source copy of the tarball file
     # @param [string] dest_pe_tarball_path Path to put the tarball at on the target host
-    # @param [string] target_master Host to upload to
     #
     # @return [true,false] Based on exit status of the bolt task
-    def upload_pe_tarball(src_pe_tarball_path, dest_pe_tarball_path = TMP_WORK_DIR, \
-                          target_master = @target_master)
+    def upload_pe_tarball(src_pe_tarball_path, dest_pe_tarball_path = TMP_WORK_DIR)
```

Comment:
I think ditch the dest_pe_tarball_path as well.  We don't really want to support putting it anywhere but TMP_WORK_DIR.  That will make future work easier.